aiobotocore/endpoint.py

```diff
@@ -72,24 +93,28 @@ class ClientResponseProxy:
     """Proxy object for http response useful for porting from
     botocore underlying http library."""
 
+    # NOTE: unfortunately we cannot inherit from ClientReponse because it also
+    # uses the `content` property
     def __init__(self, *args, **kwargs):
-        self._impl = ClientResponse(*args, **kwargs)
-        self._body = None
+        self._response = ClientResponse(*args, **kwargs)
```

Comment:
This even better! Later we can figure out what exact methods and  properties are used and only proxy those.